apps/cli-gui/src/app/app.routes.ts

```diff
@@ -1,37 +1,12 @@
 import { Routes } from '@angular/router';
 
-import { currentWorkspaceGuard } from './guards/current-workspace.guard';
 
 export const APP_ROUTES: Routes = [
   {
     path: '',
     title: 'Home',
-    canActivate: [currentWorkspaceGuard],
-    children: [
-      {
-        path: '',
-        redirectTo: 'generators',
-        pathMatch: 'full',
-      },
-      {
-        path: 'generators',
-        loadChildren: () => import('./generators/generators.routes'),
-      },
-      {
-        path: 'configuration',
-        loadComponent: () =>
-          import('@angular-cli-gui/configuration').then(
-            (m) => m.ConfigurationComponent
-          ),
-      },
-      {
-        path: 'executors',
-        loadComponent: () =>
-          import('@angular-cli-gui/executors').then(
-            (m) => m.ExecutorsComponent
-          ),
-      },
-    ],
+    // canActivate: [currentWorkspaceGuard],
+    loadChildren: () => import('./home/home.routes').then((m) => m.HOME_ROUTES),
```

Comment:
if you do `export default HOME_ROUTES` you could change here to
```suggestion
    loadChildren: () => import('./home/home.routes'),
```

Reply:
nice, changed and retrieved the canActivate